src/Datamap.js

```diff
@@ -0,0 +1,279 @@
+import d3 from 'd3';
+import topojson from 'topojson';
+import Datamaps from 'datamaps/dist/datamaps.usa.min.js';
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+class Datamap extends Component {
```

Comment:
I think this component is named wrongly (as @wfro hinted at below). Datamap is the singular of Datamaps but that's not really what this component is doing. It's a US state map or you could call it a Red Blue state map or something like that. Name the thing you're building after what it does not the pieces it's built out of.